### PR TITLE
Portal Automation

### PR DESCRIPTION
**Pull request by saravanan-udayakumar**
Automation for:
1) Hurricane Module Sequence
2) Wildfire Module Sequence
3) Monitor My World & Graph Interactive - Linked

### CODE REVIEW

File: cypress/e2e/end-end/automatedtestsequence_hurricanemodule.spec.js [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+                username: C[eachStudent + '_USERNAME'],
+                firstName: C[eachStudent + '_FIRSTNAME'],
+                lastName: C[eachStudent + '_LASTNAME'],
+                password: C[eachStudent + '_PASSWORD'],
+            }
+            studentIndex++;
+            cy.login(studentDetails.username, studentDetails.password);
+            BTN_ACTIVITY_RUN(CLASS_NAME).click();
+            for(let activityIndex = 2 ; activityIndex <= totalActivitiesInSequence - 4; activityIndex++){
+                cy.log("For Loop Inside");
```
Comment (tejal-shah) [inline cypress/e2e/end-end/automatedtestsequence_hurricanemodule.spec.js:66]: @saravanan-udayakumar , can we please remove this cy.log()? It was probably added for debugging.

File: cypress/e2e/end-end/automatedtestsequence_hurricanemodule.spec.js [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+                let currentActivity = automatedtestSequenceHurricaneModuleData[activityIndex];
+                let activityName = currentActivity.name;
+                cy.log("Lara Sequence Helper");
```
Comment (tejal-shah) [inline cypress/e2e/end-end/automatedtestsequence_hurricanemodule.spec.js:69]: And maybe this one too?

File: cypress/e2e/end-end/automatedtestsequence_hurricanemodule.spec.js [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+                username: C[eachStudent + '_USERNAME'],
+                firstName: C[eachStudent + '_FIRSTNAME'],
+                lastName: C[eachStudent + '_LASTNAME'],
+                password: C[eachStudent + '_PASSWORD'],
+            }
+            studentIndex++;
+            cy.login(studentDetails.username, studentDetails.password);
+            BTN_ACTIVITY_RUN(CLASS_NAME).click();
+            for(let activityIndex = 2 ; activityIndex <= totalActivitiesInSequence - 4; activityIndex++){
+                cy.log("For Loop Inside");
```
Comment (tejal-shah) [inline cypress/e2e/end-end/automatedtestsequence_hurricanemodule.spec.js:99]: Another cy.log() to be removed.

File: cypress/e2e/end-end/automatedtestsequence_hurricanemodule.spec.js [hunk 4]
```diff
@@ -0,0 +1,192 @@
...
+                let currentActivity = automatedtestSequenceHurricaneModuleData[activityIndex];
+                let activityName = currentActivity.name;
+                cy.log("Lara Sequence Helper");
```
Comment (tejal-shah) [inline cypress/e2e/end-end/automatedtestsequence_hurricanemodule.spec.js:102]: Another cy.log() to be removed.

File: cypress/e2e/end-end/automatedtestsequence_wildfiremodule.spec.js [hunk 5]
```diff
@@ -0,0 +1,194 @@
...
+                firstName: C[eachStudent + '_FIRSTNAME'],
+                lastName: C[eachStudent + '_LASTNAME'],
+                password: C[eachStudent + '_PASSWORD'],
+            }
+            studentIndex++;
+            cy.login(studentDetails.username, studentDetails.password);
+            // BTN_ACTIVITY_RUN(CLASS_NAME).click();
+            BTN_ACTIVITY_RUN(CLASS_NAME).click();
+            for(let activityIndex = 2 ; activityIndex <= totalActivitiesInSequence - 4; activityIndex++){
+                cy.log("For Loop Inside");
```
Comment (tejal-shah) [inline cypress/e2e/end-end/automatedtestsequence_wildfiremodule.spec.js:67]: Another cy.log() to be removed.

File: cypress/e2e/end-end/automatedtestsequence_wildfiremodule.spec.js [hunk 6]
```diff
@@ -0,0 +1,194 @@
...
+                let currentActivity = automatedtestSequenceWildFireModuleData[activityIndex];
+                let activityName = currentActivity.name;
+                cy.log("Lara Sequence Helper");
```
Comment (tejal-shah) [inline cypress/e2e/end-end/automatedtestsequence_wildfiremodule.spec.js:70]: Another cy.log() to be removed.

File: cypress/e2e/end-end/automatedtestsequence_wildfiremodule.spec.js [hunk 7]
```diff
@@ -0,0 +1,194 @@
...
+                firstName: C[eachStudent + '_FIRSTNAME'],
+                lastName: C[eachStudent + '_LASTNAME'],
+                password: C[eachStudent + '_PASSWORD'],
+            }
+            studentIndex++;
+            cy.login(studentDetails.username, studentDetails.password);
+            // BTN_ACTIVITY_RUN(CLASS_NAME).click();
+            BTN_ACTIVITY_RUN(CLASS_NAME).click();
+            for(let activityIndex = 2 ; activityIndex <= totalActivitiesInSequence - 4; activityIndex++){
+                cy.log("For Loop Inside");
```
Comment (tejal-shah) [inline cypress/e2e/end-end/automatedtestsequence_wildfiremodule.spec.js:101]: Another cy.log() to be removed.

File: cypress/e2e/end-end/automatedtestsequence_wildfiremodule.spec.js [hunk 8]
```diff
@@ -0,0 +1,194 @@
...
+                let currentActivity = automatedtestSequenceWildFireModuleData[activityIndex];
+                let activityName = currentActivity.name;
+                cy.log("Lara Sequence Helper");
```
Comment (tejal-shah) [inline cypress/e2e/end-end/automatedtestsequence_wildfiremodule.spec.js:104]: Another cy.log() to be removed.

**Review by tejal-shah — APPROVED** [review]
LGTM 👍 Just added a few comments for removing a few debug cy.log() comments.

**Comment by saravanan-udayakumar** [comment]
> LGTM 👍 Just added a few comments for removing a few debug cy.log() comments.

@tejal-shah, Thank You. Removed all the debug cy.log() comments.

**Pull request merged**